docs/crio.conf.5.md

```diff
@@ -167,8 +167,8 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
 **container_attach_socket_dir**="/var/run/crio"
   Path to directory for container attach sockets.
 
-**bind_mount_prefix**=""
-  The prefix to use for the source of the bind mounts.
+**bind_mount_prefix**="prefix"
```

Comment:
The new docs validation checks the value too. I guess it should be removed here. :)